pixart.js

```diff
@@ -1 +1,38 @@
 
+var color = document.getElementById("color-field");
+
+document.getElementById("set-color").addEventListener("click", function(evt){
+	evt.preventDefault();
+	document.getElementsByClassName("brush")[0].style.backgroundColor = color.value
+	console.log("Yay, you just typed")
+
+})
+
+    for(var x = 0; x < 8000; x++) {
+      var addSquare = document.createElement("div");
+      document.body.appendChild(addSquare)
+      addSquare.className = "square";
+    }
+
+    var square = document.getElementsByClassName("square");
```

Comment:
Make sure to make pluralization of variable names match. Since this holds an array of squares, I'd call it `squares`.